benchmark_runner/common/oc/oc.py

```diff
@@ -1534,6 +1534,35 @@ def extract_vm_results(self, vm_name: str = '', start_stamp: str = '', end_stamp
                             results_list.append(line.strip().split(':')[data_index:])
         return results_list
 
+    @typechecked
+    @logger_time_stamp
+    def generate_must_gather(self, destination_path: str = '/tmp'):
+        """
+        Generates OpenShift must-gather logs based on the OCP version and stores it in the destination path.
```

Comment:
The collection procedure isn't based on the OCP version.  Why is it needed here?  Not that big of a deal (I'm going to approve it anyway), but that's being emphasized in the comment.